.idea/gradle.xml

```diff
@@ -20,7 +20,8 @@
           </set>
         </option>
         <option name="resolveModulePerSourceSet" value="false" />
+        <option name="useQualifiedModuleNames" value="true" />
```

Comment:
please don't change anything in this file.

Reply:
ok, I accidentally pushed

Reply:
@FareesHussain to make sure the suggested changes are done, the author of the review requested will be the one who resolve these comments. What best the PR author can do is make some comments like done if it is done or anything if needs a conversation ahead. Please don't resolve such comments by yourself.

Reply:
ok

Reply:
Resolving this on behalf of @anandwana001